test/internal/k8sutils/utils.go

```diff
@@ -54,12 +54,12 @@ func MustGetClientset() (*kubernetes.Clientset, error) {
 	return clientset, nil
 }
 
-func MustGetRestConfig(t *testing.T) *rest.Config {
+func MustGetRestConfig() (*rest.Config, error) {
```

Comment:
The `Must` prefix is a common idiom in Go to denote a panic will occur instead of returning an error. Can you change this to `GetRestConfig`?

Reply:
Yeah I can change it, it's not used anywhere but test code

Reply:
I'll change it back

Reply:
Well actually, it's the only method in here, that receives `t *testing.T` as an argument, which I thought was odd
So I'm inclined to rename it `GetRestConfig`, but still leave out the `t *testing.T` argument